src/app/chat/chat-list/chat-list.component.html

```diff
@@ -11,29 +11,78 @@
     </div>
   </ng-container>
   <ion-list class="desktop-view ion-no-padding">
-    <!-- cohort chat -->
-    <ion-item [lines]="'none'" class="create-channel-container">
+    <!-- announcement chats -->
+    <ion-item [lines]="'full'" color="light-grey-75" class="create-channel-container">
+      <ion-icon class="section-icon" color="ocean" name="megaphone" slot="start"></ion-icon>
+      <ion-button class="arrow-button" slot="start" fill="clear" size="small" (click)="expandAndShrinkMessageSections('announcement')">
```

Comment:
Just simply do `isAnnouncementChatExpand = ! isAnnouncementChatExpand` for the click event. Don't need to create a function only for this.
Same for others